server/routes/v1/user.ts

```diff
@@ -129,11 +130,21 @@ async function resetPassHandler(req: Request, res: Response) {
   }
 }
 
+async function me(req: Request, res: Response) {
+  try {
+    const user = getAuthUser(res);
+    res.status(200).json(user);
+  } catch (err) {
+    console.error(err);
+  }
+}
+
 userRouter.post("/signin", signInHandler);
 userRouter.post("/signup", signUpHandler);
 userRouter.post("/signout", signOut);
 
 userRouter.get("/confirm", confirmEmailHandler);
 userRouter.get("/password-reset", resetPassHandler);
+userRouter.get("/me", me);
```

Comment:
I think this needs to go through the auth middleware before calling me

Reply:
^

Reply:
#42